aspnetcore/security/authentication/community.md

```diff
@@ -6,11 +6,12 @@ This page contains community provided open source authentication options for ASP
 
 The list below is sorted alphabetically.
 
-| Name                                                                                                                  | Description                                                                                                                                         |
-|:----------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
-| [AspNet.Security.OpenIdConnect.Server (ASOS)](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) | Low-level/protocol-first OpenID Connect server framework for ASP.NET Core and OWIN/Katana                                                           |
-| [IdentityServer4](https://identityserver.io/)                                                                         | OpenID Connect and OAuth 2.0 framework for ASP.NET Core - officially certified by the OpenID Foundation and under governance of the .NET Foundation |
-| [OpenIddict](https://github.com/openiddict/openiddict-core)                                                           | Easy-to-use OpenID Connect server for ASP.NET Core                                                                                                  |
-| [PwdLess](https://github.com/pwdless/pwdless)                                                                         | Simple, stateless, passwordless authentication for ASP.NET Core                                                                                       |
+| Name | Description |
+|:--------------|:------------------|
+| [AspNet.Security.OpenIdConnect.Server (ASOS)](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) | Low-level/protocol-first OpenID Connect server framework for ASP.NET Core and OWIN/Katana |
+| [Auth0](https://auth0.com/) | Cloud OAuth provider  |
```

Comment:
@Rick-Anderson 

I think instead of "Cloud OAuth provider", please say "Cloud Identity Provider"